CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [unreleased]
 
+Fix: resize font for tiny `TextInputWithUnit` component.
```

Comment:
Je propose quelque chose d'un poil plus précis : "Change font-size on `--tiny` modifier for `TextInputWithUnit` component." Qu'est-ce que tu en penses ?